MediaGalleryRenditions/Test/Integration/Model/GetRenditionPathTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryRenditions\Test\Integration\Model;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Filesystem;
+use Magento\Framework\Filesystem\Directory\WriteInterface;
+use Magento\Framework\Filesystem\Directory\ReadInterface;
+use Magento\MediaGalleryRenditionsApi\Api\GetRenditionPathInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class GetRenditionPathTest extends TestCase
+{
+    private const MEDIUM_SIZE_IMAGE = '/magento_medium_image.jpg';
+
+    private const LARGE_SIZE_IMAGE = '/magento_large_image.jpg';
+
+    /**
+     * @var GetRenditionPathInterface
+     */
+    private $getRenditionPath;
+
+    /**
+     * @var ReadInterface
+     */
+    protected static $mediaDir;
+
+    public static function setUpBeforeClass(): void
+    {
+        /** @var WriteInterface $mediaDirectory */
+        $mediaDirectory = Bootstrap::getObjectManager()->get(
+            Filesystem::class
+        )->getDirectoryWrite(
+            DirectoryList::MEDIA
+        );
+
+        self::$mediaDir = $mediaDirectory->getAbsolutePath();
+
+        $fixtureDir = realpath(__DIR__ . '/../_files');
+
+        copy($fixtureDir . self::LARGE_SIZE_IMAGE, self::$mediaDir . self::LARGE_SIZE_IMAGE);
```

Comment:
I'd copy the file directly in the test. That way the only source of image path is going to be dataprovider and the constants can be removed.

Also please use the `\Magento\Framework\Filesystem\DriverInterface` for copying files. 

```
        $absolutePath = realpath(__DIR__ . '/../../_files/' . $path);
        $mediaAbsolutePath = $this->mediaDirectory->getAbsolutePath($path);
        $this->driver->copy(
            $absolutePath,
            $mediaAbsolutePath
        );
```

Also, I think it's better to move the test images one directory up to the `Test` directory as they can be potentially reused by other types of tests